app/templates/auth/create-user.html

```diff
@@ -105,7 +102,7 @@ <h3 class="validation-masthead-heading" id="validation-masthead-heading">
             {%
               with
               type = "save",
-              label = "Create account" if role == 'buyer' else "Create contributor account"
+              label = "Create account" if role == 'buyer' else "Save and continue"
```

Comment:
Why "Save and continue" for suppliers? This the final step for account creation in both cases, so "Create account" should make sense in both cases. Or are there some new additional supplier steps after this one?

Reply:
This was not my decision. See story on trello.

Reply:
@constancecerf

Reply:
If you are going from the create a supplier journey, users think they've already created the account and now they are just creating a login to activate it. So the other option would be to say either 'Complete account' or 'Confirm account'

Reply:
I've just had a chat with Karl, and @risicle, we can just be consistent with what's already in the code as long as it makes sense here.

Reply:
@constancecerf so can we keep "Create account" for both buyers and suppliers?